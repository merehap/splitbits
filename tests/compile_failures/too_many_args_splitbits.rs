use splitbits::*;

fn main() {
    splitbits!(min=u8, 0b11011101, "aaabbccc", whatisthisthing);
}
