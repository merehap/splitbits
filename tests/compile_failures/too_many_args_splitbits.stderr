error: proc macro panicked
 --> tests/compile_failures/too_many_args_splitbits.rs:4:5
  |
4 |     splitbits!(min=u8, 0b11011101, "aaabbccc", whatisthisthing);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: splitbits must take at most three arguments: a precision, then an input value, then a template. Found:
          `min=u8, 0b11011101, "aaabbccc", whatisthisthing`
